source/javascripts/core/BitriseYml.ts

```diff
@@ -12,48 +9,4 @@ type Meta = Required<BitriseYml>['meta'] & {
   };
 };
 
-function deleteWorkflow(yml: BitriseYml, workflowId: string): BitriseYml {
```

Comment:
Why is this deleted? Was it unused?

Reply:
Yes, it is unused. We're using the deleteWorkflow function in models/BitriseYmlService.ts. It hasn't been decided how we'll organize things yet, but everything will be in the right place (hopefully) before we release the new React-only Workflows page. 🙂

Reply:
Additionally, this file had an import cycle before the deletion...